README.md

```diff
@@ -1,13 +1,19 @@
-Demo Express app for projects participating in Exchange Discover program
+A demo Express app for projects participating in xExchange Discover program.
 
 ## Quick start
 
-1. Run `npm install` in the project directory
-2. Run `npm start` to start the app on port 3000 (this can be changed in `server.js`)
+1. Run `npm install` to install the app.
+2. Run `npm run create-env-file` to generate a `.env` configuration file. 
```

Comment:
We could also not add the script and just tell to copy `.env.example` into `.env` and to fill the variables inside. Might remove some magic.